CMakeLists.txt

```diff
@@ -152,8 +161,26 @@ if(OPT_DEBUG)
 endif()
 
 # ManyMouse
-set(C_MANYMOUSE OFF)    # TODO
-set(SUPPORT_XINPUT2 OFF)    # TODO
+if (DOSBOX_PLATFORM_MACOS)
+  include (${CMAKE_CURRENT_SOURCE_DIR}/scripts/cmake/FindIOKit.cmake)
+  set(C_MANYMOUSE (${HAVE_FD_ZERO} AND ${IOKit_FOUND}))
+else()
+  set(C_MANYMOUSE "${HAVE_FD_ZERO}")
+endif()
```

Comment:
So it will be always on? Don't we want to do `OPT_MANYMOUSE`, so turn it into an option?